problemsets-mat223/modules/module7.tex

```diff
@@ -133,7 +168,30 @@
 	\]
 	Find a normal vector for $\mathcal Q$ and write $\mathcal Q$ in normal form.
 
-	XXX Finish
+	Similarly to the above example, since normal vectors $\vec n$ for $\mathcal Q$ need to be orthogonal to $\vec d_1=\mat{1\\1\\-1\\1}$, $\vec d_2=\mat{0\\1\\0\\1}$ and $\vec d_3=\mat{2\\0\\0\\0}$, we could find it by solving
```

Comment:
remove $\vec n$. Since you said "normal vector`s`" we cannot give it a name because there is more than one.

Reply:
`could find it` -> `could find the normal vectors`  (In general, avoid `it`. It is more clear to write out what `it` is supposed to stand for.)